config/routes.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+  get 'dashboard/index'
```

Comment:
Please use the 'resource' syntax.
```ruby
resources :dashboard, only: :index
```
After the change we'll have a prettier link like '/dashboard' not '/dashboard/index'